batavia/core/builtins.js

```diff
@@ -732,8 +732,9 @@ batavia.builtins.help.__doc__ = 'In Python, this is a wrapper around pydoc.help.
 batavia.builtins.hex = function(args) {
     if (args.length !== 1) {
         throw new batavia.builtins.TypeError("hex() takes exactly one argument (" + args.length + " given)");
-    }
-    return "0x" + args[0].toString(16);
+    };
+    var int = args[0].val.c[0]
```

Comment:
This doesn't print the entire integer -- it will only work for very small numbers.

Is this tested in the builtins tests?

For example,

```
>>> print(hex(0xb2594c0a7500649c618b365c0663d9d8556be9784939edb663267c0a8288))
```

Should print "0xb2594c0a7500649c618b365c0663d9d8556be9784939edb663267c0a8288", but this prints only "0x7b".